server/course/index.js

```diff
@@ -1,27 +1,52 @@
 'use strict';
 
+const { NOT_FOUND, UNAUTHORIZED } = require('http-status-codes');
 const { authorize } = require('../shared/auth/mw');
 const { Course } = require('../shared/database');
 const { createError } = require('../shared/error/helpers');
-const { NOT_FOUND, UNAUTHORIZED } = require('http-status-codes');
 const ctrl = require('./course.controller');
+const path = require('path');
 const processQuery = require('../shared/util/processListQuery')();
 const router = require('express-promise-router')();
 
+const activity = require('../activity');
+const comment = require('../comment');
+const revision = require('../revision');
+const teachingElement = require('../teaching-element');
+
+module.exports = { path: '/courses', router };
```

Comment:
Why is module.exports here above all the router definitions but in the above files it is not?

Reply:
I know it's not consistent, but this router has subrouters, and it has more code compared to other router files, so it's more concise to define `module.exports` above all instead of scrolling down. Also, @vladimyr wanted approach like this.

Reply:
I second @abasic on this one. Even though this is a bit different than the other ones I don't consider those reasons enough to steer away from consistency.

Reply:
> I second @abasic on this one. Even though this is a bit different than the other ones I don't consider those reasons enough to steer away from consistency.

Agreed, reverted to original module layout.